### ISSUE TITLE
[suggestion] Compacting Drawers with dusts

### ISSUE THREAD

**Issue by JasonMcRay** [jaquadro/StorageDrawers#172]
Would it be possible to add support for Dusts (GT and IC2) as well for the Compacting Drawers? They can be crafted into 4 Small Piles or 9 Tiny Piles.

It is shaped crafting. if the full dust is on the left side it gives 4x Small Piles and if on the left side of the crafting grid it gives 9x Tiny Piles.

**Comment by jaquadro** [comment]
I guess I'll need to look at the actual recipe entries to see what's involved.

They could of course be added manually with the MT API (https://github.com/jaquadro/StorageDrawers/wiki/MineTweaker-API)

**Comment by JasonMcRay** [comment]
Yes. i know I can add them manually... but... well... adding over 500 dusts is quite a lot
